lesson2/src/server.ts

```diff
@@ -0,0 +1,96 @@
+import http from 'http';
+
+import { PORT } from './config';
+import logger from './logger';
+
+function getRawBody(req: http.IncomingMessage): Promise<string> {
+  const chunks: Uint8Array[] = [];
+  return new Promise((resolve, reject) => {
+    req
+      .on('data', (chunk) => {
+        chunks.push(chunk);
```

Comment:
You can use async iterators without subscribing on different event
```js
for await (const chunk of req) {
      body.push(chunk);
 }
```

Reply:
Thanks for the advice. It looks much cleaner now!